CMakeLists.txt

```diff
@@ -1192,31 +1189,19 @@ endif()
 #       - Secure renegotiation
 #       - Fallback SCSV
 
+add_option(WOLFSSL_OCSP "Enable OCSP (default: disabled)" "no" "yes;no")
+add_option(WOLFSSL_OCSPSTAPLING "Enable OCSP Stapling" "no" "yes;no")
```

Comment:
Could you add the default to each of the help strings for this and stapling v2?

Reply:
Done